docker-compose-k6.account.yaml

```diff
@@ -32,8 +31,7 @@ services:
       - redis
       - ipfs
     environment:
-      PROVIDER_BASE_URL: 'http://account-service-webhook:3001/webhooks/account-service'
-      WEBHOOK_BASE_URL: 'http://account-service-webhook:3001/webhooks'
```

Comment:
Note: I didn't see where `WEBHOOK_BASE_URL` was used in the code, so I think it was redundant.